source/drivers/php.txt

```diff
@@ -12,33 +14,65 @@ MongoDB PHP Driver
    :depth: 1
    :class: twocols
 
-For the official MongoDB PHP Driver reference, see:
+Introduction
+------------
+
+The currently maintained driver is the `mongodb
+<https://pecl.php.net/package/mongodb>`_ extension available from PECL.
+
+You might consider using the driver with the complimentary `PHP library
+<https://github.com/mongodb/mongo-php-library>`_, which implements a more
+full-featured API on top of the bare-bones driver.
 
-- `MongoDB PHP Library Documentation
-  <https://docs.mongodb.com/php-library/>`_
+Further information on this architecture may be found 
+in the `PHP.net documentation <http://php.net/manual/en/mongodb.overview.php>`_.
 
-- `MongoDB PHP Extension Documentation
-  <http://php.net/manual/en/set.mongodb.php>`_
+- `Tutorials <https://www.php.net/manual/en/mongodb.tutorial.php>`__
 
-Drivers
--------
+- `Driver Architecture and Internals <https://www.php.net/manual/en/mongodb.architecture.php>`__
+
+- `API Reference <https://php.net/manual/en/book.mongodb.php>`_
+
+- `Changelog <https://github.com/mongodb/mongo-php-driver/releases>`__
+
+- `Source Code <https://github.com/mongodb/mongo-php-driver>`__
+
+
+Installation
+------------
+
+You can install the driver using 
+`PECL <https://www.php.net/manual/en/mongodb.installation.pecl.php>`_:
+
+.. code-block:: sh
+
+   $ pecl install mongodb
+   $ echo "extension=mongodb.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
+
+The preferred method of installing the optional but recommended PHP library is with 
+`Composer <https://getcomposer.org/>`_ by running the following from your project root:
+
+.. code-block:: sh
+
+   $ composer require mongodb/mongodb
+
+Additional installation instructions may be found in the
+`library documentation <https://docs.mongodb.com/php-library/current/tutorial/install-php-library/>`_.
+
+
+Connect to MongoDB Atlas
+------------------------
+
+To connect to a `MongoDB Atlas <https://docs.atlas.mongodb.com/>`_ cluster, use the `Atlas connection string <https://docs.atlas.mongodb.com/driver-connection>`__ for your cluster:
+
+.. code-block:: php
```

Comment:
I wonder why this code block isn't getting any syntax highlighting

Reply:
Seems I needed the `<?php` tag.